WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/ProductFormViewModel+ObservablesTests.swift

```diff
@@ -201,14 +201,17 @@ final class ProductFormViewModel_ObservablesTests: XCTestCase {
         XCTAssertEqual(updatedUpdateEnabled, false)
     }
 
-    func testObservablesFromUploadingAnImage() {
-        // Arrange
+    func test_observables_from_mockProductImageUploader_when_uploadMediaAssetToSiteMediaLibrary_then_returns_true() {
```

Comment:
I would make this function name more meaningful:
- I don't get what's the point of`from_mockProductImageUploader` :)
- The condition `when_uploadMediaAssetToSiteMediaLibrary` could have more information.
- I don't see what returns true 
Furthermore, it seems like we are adding several test cases into one function, and we are testing other things apart from the view model. It makes this test function a bit confusing.

But if we want to keep as it is now, what about a name like
`func test_observables_when_productImageActionHandler_uploads_media_asset_then_observables_are_called
` ?

Reply:
Definitely, that naming is much more close to what this is trying to do 🙇 . Changed here: 0ac2ceeb0b3d623b60845d7210cedcc751550a92